_includes/selector.html

```diff
@@ -427,8 +427,7 @@
             getAdditionalPkgName(pkg) {
                 var dask_prefix = this.active_release === "Nightly" ? "dask/label/dev::" : "";
```

Comment:
@charlesbluca We have a variable `dask_prefix` in a couple places that appears to be unused now. Can we delete that?

Reply:
Pushed a change to Charles' PR to do that: https://github.com/rapidsai/docs/pull/521/commits/d1da50494ef95daec0c82547f18434dca074a545